pkg/constants/availability_zone.go

```diff
@@ -0,0 +1,29 @@
+package constants
+
+type AvailabilityZone string
+
+func (az AvailabilityZone) Name() string {
+	return string(az)
+}
+
+const (
```

Comment:
better NOT to use consts here; rkv targets to work in other cloud env.

Reply:
The reason to add constants here is to validate if the availability zone is valid since it will be used in hashing. If we use other cloud envs, we can add more constants.